hail_search/test_search.py

```diff
@@ -13,3 +292,23 @@ async def test_status(self):
             self.assertEqual(resp.status, 200)
             resp_json = await resp.json()
         self.assertDictEqual(resp_json, {'success': True})
+
+    async def test_search(self):
+        search_body = get_hail_search_body(sample_data=FAMILY_2_VARIANT_SAMPLE_DATA)
+        async with self.client.request('POST', '/search', json=search_body) as resp:
+            self.assertEqual(resp.status, 200)
+            resp_json = await resp.json()
```

Comment:
Can we test the search class directly without going through the web server? So that no synchronization is needed for the search tests.

Reply:
But that would be a substantially worse test suite. We don't care at all how the search class behaves, we only care about how the actual API requests that seqr will be using behave. In the main seqr repo we have been slowly phasing out testing utility functions directly in favor of testing them through API endpoints for that exact reason, and in fact we have had bugs in the past couple months that slipped through because we tested the function directly and not through the API. The async behavior here is part of the well supported `AioHTTPTestCase` and adds almost no time to the test suite as the fake responses are pretty quick